test/test_flake8.py

```diff
@@ -11,13 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from ament_flake8.main import main
 import pytest
 
+from flake8.api.legacy import get_style_guide
```

Comment:
Why is the new import in a different location than the previous one? Based on the naming I would expect it to be the same.